api/v1/users/PKPUserHandler.inc.php

```diff
@@ -249,6 +249,8 @@ private function _buildListRequestParams($slimRequest) {
 			}
 		}
 
+		$returnParams['contextId'] = $context->getId();
```

Comment:
I think this API endpoint can be called site-wide, which would cause a fatal error on non-object here (as `$context` would be `null`).